packages/blade/src/components/Divider/Divider.native.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components/native';
```

Comment:
Instead of having `Divider.native` & `Divider.web` can we have a single `Divider` file and use `BaseBox` within it. BaseBox has `borderColor`, `borderStyle`, `width` & `height`

Reply:
This will simplify the complexity of code for Divider

Reply:
done, however -
- could not find `borderStyle` on `BaseBox` (have used `style` prop for now). Please let me know if missed something.
- default `borderWidth` supports `none, thick, and thin`. However we also have a `thinner` variant here. Have also used style prop for that.
- Is it safe to assume that Divider will always be rendered inside a flex container -
    - If yes, then for vertical divider to take the full height have added `alignSelf: stretch` property.
    - If no, then how do we determine (to also add `height: 100%`) whether parent container is flex of not?
        - Tried to explore a bit, MUI uses a boolean prop `flexItem` for this scenario 🤔?

Reply:
done, however - 
- the exisiting `borderWidth` allows `none, thin, and thick` but in our usecase there is a `thinner` option as well.
- also could not find `border style` prop on `baseBox`. Please let me know if missed something.
- for both above properties have used `style` prop
    - had tried using shorthand notation `borderLeft: ${borderStyle} ${width}` to avoid `style` prop but borderStyle was getting overridden with `solid`
- Also can we assume that Divider will always be render within a flex container?
    - if no, how do we determine whether parent container is flex? (ref [flexItem prop on MUI divider](https://mui.com/material-ui/api/divider/))
    - for now have used `flexGrow` and `alignItems` to auto adjust the width and height of divider.
    - let's please have this discussion?